src/seekcentral.cpp

```diff
@@ -593,7 +616,8 @@ namespace Sleipnir {
                          true, //check dataset size (to be overwritten)
                          NULL,  // gsl_rng
                          useNibble,
-                         settings.numThreads);
+                         settings.numThreads,
+                         0.5);  // min percent datasets
```

Comment:
Maybe we should add a SeekConfig parameter also.

Reply:
Would this be in SeekSettings?

Reply:
We could add to SeekSettings, and optionally also add to the thrift SeekQueryParams.